bibliopixel/drivers/SPI/APA102.py

```diff
@@ -4,7 +4,12 @@
 
 
 class APA102(SPIBase):
-    """Driver for APA102/SK9822 based LED strips on devices like the Raspberry Pi and BeagleBone"""
+    """Driver for APA102/SK9822 based LED strips on devices like
```

Comment:
For multiline comments, I tend to prefer comments written like this:

    """
    Your comment here...
    etc.
    """

where the first and last `"""` are on a line by themselves.  It reads neater, and it's easier to edit...!

We seem to do both, and [PEP 257](https://www.python.org/dev/peps/pep-0257/) also allows both.